### DISCUSSION TITLE
How to close current TAB?

### DISCUSSION THREAD

**Discussion by tonmoy998** [LazyVim/LazyVim#1292] [Q&A] [answered]
Is there any key binding to close Current Tab without quiting nevim?

**Comment by dpetka2001** [comment]
It depends. If it's the last tab I don't think that's possible. Because the buffers of the open files that exist, must be viewed in at least one tab. So when Neovim starts, it creates automatically one tab with a window in it and you open a buffer inside that window. You can read more about it in [Neovim docs](https://neovim.io/doc/user/windows.html#windows-intro).
If it's not the last tab, then you just `:q` to close the current tab.

**Reply by tonmoy998** [reply]
Thank you for the information

**Comment by skoch13** [comment]
Lazyvim has built-in keybinding for that: leader + TAB + d

**Reply by tonmoy998** [reply]
Can't close the last tab

**Reply by skoch13** [reply, marked as answer]
Then you want to close buffers, not a tab.

**Reply by tonmoy998** [reply]
Thank you man 🙏❤️
